Bottom: 52d02cb4f737e5c42017e03749150023b8a90ca3
Top:    93bffa73ac859aff8a748b67bcfb5276515668cf
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2021-11-01 16:35:22 +0200

hack poky.conf to get rid of snapshot -> 3.4.66

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/meta-poky/conf/distro/poky.conf b/meta-poky/conf/distro/poky.conf
index 2dc3606ae5..450516d45e 100644
--- a/meta-poky/conf/distro/poky.conf
+++ b/meta-poky/conf/distro/poky.conf
@@ -1,7 +1,9 @@
 DISTRO = "poky"
 DISTRO_NAME = "Poky (Yocto Project Reference Distro)"
-DISTRO_VERSION = "3.4+snapshot-${METADATA_REVISION}"
-DISTRO_CODENAME = "master"
+#DISTRO_VERSION = "3.4+snapshot-${METADATA_REVISION}"
+DISTRO_VERSION = "3.4.66"
+#DISTRO_CODENAME = "master"
+DISTRO_CODENAME = "master-as-honister"
 SDK_VENDOR = "-pokysdk"
 SDK_VERSION = "${@d.getVar('DISTRO_VERSION').replace('snapshot-${METADATA_REVISION}', 'snapshot')}"
 SDK_VERSION[vardepvalue] = "${SDK_VERSION}"
